third_party/unifex.BUILD

```diff
@@ -35,6 +37,11 @@ cc_library(
     name = src[:-4],
     size = "small",
     srcs = [src],
+    target_compatible_with = select({
+        "@platforms//os:linux": [],
+        "@platforms//os:windows": [],
+        "//conditions:default": ["@platforms//:incompatible"],
```

Comment:
Probably only mark this as incompatible with macos, and a comment about why? Looks like they have a macos-build in CI, so I was expecting it to work, but I haven't looked into it at all.